crates/biome_project/src/node_js_project/tsconfig_json.rs

```diff
@@ -0,0 +1,20 @@
+use crate::{LanguageRoot, Manifest};
+use biome_deserialize::json::deserialize_from_json_ast;
+use biome_deserialize::Deserialized;
+use biome_deserialize_macros::Deserializable;
+use biome_json_syntax::JsonLanguage;
+use rustc_hash::FxHashMap;
+
+#[derive(Debug, Default, Clone, Deserializable)]
```

Comment:
If you want to reproduce your previous handmade impl, you have to add `#[deserializable(unknown_fields = "allow")]` to allow ignoring unknown fields.

```suggestion
#[derive(Debug, Default, Clone, Deserializable)]
#[deserializable(unknown_fields = "allow")]
```